src/averageAges.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+// callback for average age from first two tasks
+const calcAvg = (acc, curr) => {
+  return acc + curr.died - curr.born;
+};
```

Comment:
let's make it an actual `calcAvg`, so we provide it with an array of people
and get the average on `return`:
```
const calcAvg = (people) => {
  return people.reduce((acc, {died, born}) => acc + died - born, 0) / people.length;
};
```